website/siteConfig.js

```diff
@@ -87,6 +87,7 @@ const siteConfig = {
   title: 'Jest',
   tagline: '🃏 Delightful JavaScript Testing',
   url: 'https://facebook.github.io',
+  cname: 'jestjs.io',
```

Comment:
This will fix the CNAME being deleted on deploy

Reply:
Does this fix #6255?